### PR TITLE
Remove almost-empty helper module; shuffle Html; ...

### LINKED ISSUE #180: Resolve module Api.
There is only one function in there.  It could go to `Persistent.Api` for now?

### PR DESCRIPTION
**Pull request by fisx**
Fixes #180.

### CODE REVIEW

**Comment by fisx** [comment]
Hum, not sure about the changes around `Category` any more.  I guess we can merge and change later, but I need to get some advice on browser hacking to figure out how we want to do this.  Not working: category selection in `CreateIdea`; category rendering in `ViewIdea`.

**Comment by np** [comment]
LGTM

File: src/Frontend/Page/Idea.hs [hunk 1]
```diff
@@ -181,20 +177,10 @@ instance FormPage CreateIdea where
                             div_ [class_ "category-radios"] $ do
                                 DF.inputRadio True "idea-category" v
                             div_ [class_ "icon-list m-inline category-image-select"] $ do
-                                ul_ $ do
+                                ul_ $ toHtml `mapM_` [(minBound :: Category)..]
```
Comment (andorp) [inline src/Frontend/Page/Idea.hs]: Please introduce a newtype for this, converting a category value to special list item only for this module, in my opinion has to be in a newtype.

**Comment by andorp** [comment]
LGTMITIG

**Pull request merged**